catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -289,10 +251,8 @@ public void setAttributeOverrides(List<String> attributeOverrides) {
             "Invalid attribute override configured for monitored directory");
       }
 
-      if (attributeOverrideMap.get(keyValue[0]) == null) {
-        attributeOverrideMap.put(keyValue[0], new ArrayList<String>());
-      }
-      ArrayList valueList = (ArrayList) attributeOverrideMap.get(keyValue[0]);
+      attributeOverrideMap.computeIfAbsent(keyValue[0], k -> new ArrayList<String>());
```

Comment:
```suggestion
      final List valueList = attributeOverrideMap.computeIfAbsent(keyValue[0], k -> new ArrayList<String>());
```

Reply:
☝️ There is no need to access the map twice. `computeIfAbsent()` will return you either the existing value or the new one.